AEPCore/Sources/eventhub/EventType.swift

```diff
@@ -41,6 +41,6 @@ public class EventType: NSObject {
     public static let genericPii = "com.adobe.eventType.generic.pii"
     public static let genericData = "com.adobe.eventType.generic.data"
     public static let edge = "com.adobe.eventType.edge"
-    public static let consent = "com.adobe.eventType.consent"
+    public static let edgeConsent = "com.adobe.eventType.edgeConsent"
```

Comment:
I would prefer to keep this consent, but I also feel that if we do that it will not be consistent with the shared state name proposals and module name.